src/state-client.ts

```diff
@@ -27,6 +27,9 @@ export enum ServerState {
   ALL = 'all_state',
   AUTHENTICATION = 'authentication',
   CHANNEL = 'channel_state',
+  CONFIG = 'config',
+  CONFIG_CHANNEL = 'config/channel',
+  CONFIG_EXTENSION = 'config/extension',
```

Comment:
I'd prefer these to exist outside the `ServerState` enum, as they should be their own `ServerConfig` concept.